setup.py

```diff
@@ -61,7 +61,9 @@ def run(self):
     # This will let setuptools to copy ver what"s listed in MANIFEST.in
     include_package_data=True,
     install_requires=[
-        'antlr4-python3-runtime>=4.7.2', 'Jinja2>=2.10', 'ordered_set>=3.1.1',
+        # antlr issue url: https://github.com/DiODeProject/MuMoT/issues/386
```

Comment:
There is nothing wrong with Antlr, and you just followed what's suggested in that link. The comment seems unnecessary.

Reply:
I see. Then, I will just remove this comment.